productivity-suite/app/legacy-app/src/components/layout.css

```diff
@@ -32,6 +32,10 @@
   text-align: center;
 }
 
+.app-nav-list > li:has(a.active) {
+  margin-block: 0;
```

Comment:
Out of interest, why margin-block rather than margin?

Reply:
because with `margin: 0` we'd also be messing with the width of the elements and that would create some ugly movements, like so:
![margin-block](https://user-images.githubusercontent.com/61631103/201999195-5b0114a4-083e-49ef-8200-686710c818e4.gif)